web/src/beta/features/Editor/DataSourceManager/Asset/index.tsx

```diff
@@ -115,17 +111,13 @@ const Asset: React.FC<DataProps> = ({ sceneId, onSubmit, onClose }) => {
         )}
         {sourceType == "value" && (
           <>
-            <InputGroup
-              label="File Format"
-              description="File format of the data source you want to add.">
-              <Select value={fileFormat} onChange={setFileFormat}>
-                {["GeoJSON", "KML", "CZML"].map(op => (
-                  <Option key={op} value={op} label={op}>
-                    {op}
-                  </Option>
-                ))}
-              </Select>
-            </InputGroup>
+            <SelectField
+              value={fileFormat}
+              options={["GeoJSON", "KML", "CZML"].map(v => ({ key: v, label: v }))}
+              name="File Format"
+              description="File format of the data source you want to add."
```

Comment:
Sorry, I know not in scope for this PR, but can you wrap text in i18n? 🙏🏻

Reply:
Done